### DISCUSSION TITLE
Character encoding/display issues in Chrome

### DISCUSSION THREAD

**Discussion by timathom** [Jinntec/Fore#152] [General]
I have text in Russian that has been transliterated and contains diacritics:

```
<catalog yes="true" no="false" maybe="false" confidence="0" bib="811932">
  <a href="https://search.library.yale.edu/catalog/811932" target="_blank">Semʹi︠a︡ Ulʹi︠a︡novykh : tetralogii︠a︡ ; Ocherki i statʹi o Lenine : 1935-1981 / Mariėtta Shagini︠a︡n.</a>
</catalog>
```

It displays correctly in Firefox:
<img width="699" alt="Screen Shot 2022-06-30 at 10 14 38 AM" src="https://user-images.githubusercontent.com/5334524/176700158-35d7d822-038f-4232-a77c-c8a3b502a903.png">

But not in Chrome:
<img width="685" alt="Screen Shot 2022-06-30 at 10 16 04 AM" src="https://user-images.githubusercontent.com/5334524/176700377-3fedcaa0-a3f3-4e8c-a7d4-e86a8ddd40e5.png">

I've also tested with XSLTForms, and there it displays correctly in both browsers. Any ideas?

**Comment by JoernT** [comment]
@timathom such reports which clearly describe a problem can be put into a ticket directly.

regarding the issue i have no idea what might cause it. Could you provide an more complete example? What happens if you comment the Fore import?

**Reply by timathom** [reply]
Not sure what's meant by commenting the Fore import, but I just opened an issue (#153) that includes an example form.
